subsys/net/ip/Kconfig

```diff
@@ -290,6 +290,12 @@ config NET_RX_DEFAULT_PRIORITY
 	  What is the default network RX packet priority if user has not set
 	  one. The value 0 means lowest priority and 7 is the highest.
 
+config NET_ALLOW_ANY_PRIORITY
+	bool "Allow any priority to be used"
+	help
+	  If this is set, then any user given priority can be used. Otherwise the
+	  priorities are limited to 0-7 range.
```

Comment:
I think we need to emphasize that the priority is for network packets, it is not obvious now from the text.
So something like
```
bool "Allow any network packet priority to be used"
help
   If this is set, then any user given network packet priority can be used. Otherwise the
   network packet priorities are limited to 0-7 range.
```